core/dive.c

```diff
@@ -453,6 +453,20 @@ double get_weight_units(unsigned int grams, int *frac, const char **units)
 	return value;
 }
 
+int32_t altitude_to_pressure(int altitude) 		// altitude in meters above sea level
+{						// function returns atmospheric pressure in mbar
+	int i, alt;
+	static const int pressval[21] = { 1013, 984, 955, 926, 899, 872, 846, 820, 795, 771, 747, 724, 701, 679, 658, 637, 616, 597, 577, 559, 540 };
+	// Calculated atmospheric pressure values at 250m increments in altitude from sea level to 4999m,
+	// calculated following the US Standard Atmosphere 1976, US Government Printing Office, Washington DC, 1976.
+	// For intermediate altitudes, linear interpolation is performed here.
+	alt = altitude;
+	if (altitude < 0) alt = 0;
```

Comment:
Minor nit-pick: The dead sea goes down to -430 m (growing).

Reply:
Implemented this. IMPORTANT: this function cannot go to units.c because there is no such file. Unit.h comprises typedefs and simple definitions. Moving the altitude_to_pressure() function there causes untold problems. I hope my interpretation is correct?

Reply:
> Implemented this.

Nice, thank you.

> IMPORTANT: this function cannot go to units.c because there is no such file. 

Ok, perhaps we should one day generate such a file, but not now.